src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { cloneElement, HTMLAttributes } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import styled from 'styled-components';
+import { MD } from '@zendeskgarden/react-typography';
+import { ReactComponent as XStrokeIcon } from '@zendeskgarden/svg-icons/src/16/x-stroke.svg';
+import { ReactComponent as CheckLgStrokeIcon } from '@zendeskgarden/svg-icons/src/16/check-lg-stroke.svg';
+import { ReactComponent as AlertErrorStrokeIcon } from '@zendeskgarden/svg-icons/src/16/alert-error-stroke.svg';
+
+import { getColor } from '@zendeskgarden/react-theming';
+
+export interface IStyledBestPracticeTextProps {
+  hue: string;
+}
+
+export interface IFileNode {
+  name: string;
+  publicURL: string;
+}
+
+export interface IFileEdge {
+  node: IFileNode;
+}
+
+const StyledBestPracticeText = styled.div<IStyledBestPracticeTextProps>`
+  border-top: ${props => `${props.theme.borders.md} ${getColor(props.hue, 500, props.theme)}`};
+  border-bottom-left-radius: ${props => props.theme.borderRadii.md};
+  border-bottom-right-radius: ${props => props.theme.borderRadii.md};
+  background: ${props => getColor('neutralHue', 100, props.theme)};
+  padding: ${props => `${props.theme.space.md}`};
+
+  ul {
+    margin: 0;
+    list-style: none;
+  }
+
+  li {
+    padding: ${props => props.theme.space.base}px 0;
+  }
+
+  li:not(:last-child) {
+    border-bottom: ${props =>
+      `${props.theme.borders.sm} ${getColor('neutralHue', 300, props.theme)}`};
+  }
+`;
+
+const StyledIcon = styled(({ children, ...props }) =>
+  cloneElement(React.Children.only(children), props)
+)`
+  margin-right: ${props => props.theme.space.xs};
```

Comment:
Since it's fairly trivial, please follow best practice and apply RTL logic to these `website` custom components.

Reply:
> it's fairly trivial

There are many elements that don't handle RTL right now. With no other pages supporting it I don't think we should start introducing new standards until we know that RTL is a global feature that we will need.